sonar-plugin-api/src/main/java/org/sonar/api/utils/System2.java

```diff
@@ -72,7 +73,9 @@
 
   /**
    * Shortcut for {@link System#currentTimeMillis()}
+   * @deprecated since 6.6 use {@link Clock} that is available in pico.
    */
+  @Deprecated
   public long now() {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)